### PR TITLE
FLOW-419 - added in handling for en-CA in numbro

### PR DESCRIPTION
**Pull request by joshuacatlingdell**
added const in engine.ts and manually added language culter to numbro init and simpleinit rather than create new language file as we are using v 1.11 of numbro so new changes will not be collected

### CODE REVIEW

File: js/services/engine.ts [hunk 1]
```diff
@@ -882,10 +940,15 @@ export const initializeSimple = (
     }
 
     if (window.navigator.language) {
```
Comment (dshoulders) [inline js/services/engine.ts]: Would be good to extract this duplication into a function.

File: js/services/engine.ts [hunk 2]
```diff
@@ -22,6 +22,59 @@ import * as Validation from './validation';
 declare var manywho: any;
 declare var window: any;
 
+const ohCanada = {
```
Comment (dshoulders) [inline js/services/engine.ts]: Could this live in another file? Maybe a localisation service file?

File: js/services/engine.ts (hunk 1 above)
Comment (joshuacatlingdell) [reply js/services/engine.ts]: i didnt create the duplicate functions just added in the clause to use the new culture constant to the 2 existing ones. someone may want to go through and refactor that bit of the code at some time.

Comment (dshoulders) [reply js/services/engine.ts]: I think that as the duplication has grown it is more of a concern. The problem is that if an issue is identified in a section of this code and is fixed, or is improved in some way, the duplicated section could easily get missed. This probably should have been addressed when first written but as we have an opportunity to improve it now I think we should.

File: js/services/engine.ts (hunk 2 above)
Comment (dshoulders) [reply js/services/engine.ts]: This could just be a new file under services. ie. services/localisation.js.

**Comment by joshuacatlingdell** [comment]
@dshoulders updates done please recheck

**Pull request merged**